include/openPMD/Mesh.hpp

```diff
@@ -69,15 +70,25 @@ class Mesh : public BaseRecord< MeshRecordComponent >
     };  //DataOrder
 
     /**
-     * @return String representing the geometry of the mesh of the mesh record.
+     * @return Enum representing the geometry of the mesh of the mesh record.
      */
     Geometry geometry() const;
+    /**
+     * @return String representing the geometry of the mesh of the mesh record.
+     */
+    std::string geometryString() const;
     /** Set the geometry of the mesh of the mesh record.
      *
      * @param   g    geometry of the mesh of the mesh record.
      * @return  Reference to modified mesh.
      */
     Mesh& setGeometry(Geometry g);
+    /** Set the geometry of the mesh of the mesh record.
+     *
+     * @param   geometry    geometry of the mesh of the mesh record, as string
+     * @return  Reference to modified mesh.
+     */
+    Mesh& setGeometry(std::string const& geometry);
```

Comment:
Since we can move out strings now, the proper interface for this is likely by value?
```suggestion
    Mesh& setGeometry(std::string geometry);
```

Reply:
I used a const reference since that's what `Attributable::setAttribute` uses: https://github.com/openPMD/openPMD-api/blob/a5b026e861eb2906bf203915d20be30a359a9e0f/include/openPMD/backend/Attributable.hpp#L360
If we go with move semantics, we should update `setAttribute` accordingly